morpheushelper/cogs/voice_channel.py

```diff
@@ -18,6 +18,7 @@
 from models.role_voice_link import RoleVoiceLink
 from permissions import Permission
 from util import get_prefix, send_to_changelog, is_teamler
+import logging
```

Comment:
perhaps you should consider using the `getLogger` method of logging to get a specific logger and also create a logging config.
```python
logger = logging.getLogger(<name>)
```